antsibull/utils/http.py

```diff
@@ -77,7 +80,7 @@ async def __aenter__(self) -> aiohttp.ClientResponse:
             if failed:
                 break
 
-            await asyncio.sleep(retry * 0.5)
+            await asyncio.sleep(retry * wait_factor + random.uniform(0, 1))
```

Comment:
Maybe consider implementing an exponential backoff? `ansible-galaxy` does that now.

Reply:
Did that in 73c5c563e1e8cb46d4b0c8b64d2adf9a6dbf9eeb, including more randomness.